examples/oft-solana/tasks/common/config.get.ts

```diff
@@ -0,0 +1,200 @@
+import { EndpointV2 } from '@layerzerolabs/protocol-devtools-solana'
+import { OAppOmniGraph } from '@layerzerolabs/ua-devtools'
+import { createOAppFactory } from '@layerzerolabs/ua-devtools-evm'
+import {
+    OAppOmniGraphHardhatSchema,
+    SUBTASK_LZ_OAPP_CONFIG_LOAD,
+    type SubtaskLoadConfigTaskArgs,
+    TASK_LZ_OAPP_CONFIG_GET,
+} from '@layerzerolabs/ua-devtools-evm-hardhat'
+import { getReceiveConfig, getSendConfig } from '@layerzerolabs/ua-devtools-evm-hardhat'
+
+import { getSolanaReceiveConfig, getSolanaSendConfig } from './taskHelper'
+import { createSolanaConnectionFactory } from './utils'
+
+interface TaskArgs {
+    logLevel?: string
+    oappConfig: string
+}
+
+/**
+ * Helper function to determine if the point is Solana
+ * @param point {OmniPoint}
+ */
+const isSolana = (point: OmniPoint) => endpointIdToChainType(point.eid) === ChainType.SOLANA
+
+/**
+ * Helper function to get the hardhat.config.ts network name for a given endpoint id, or use the convention of
+ * networkName-environment for Solana.
+ * @param eid {EndpointId}
+ */
+const getEid = (eid: EndpointId) => {
+    switch (eid) {
+        // In the case of solana-testnet and solana-mainnet, we'll use the convention of networkName-environment
+        case EndpointId.SOLANA_V2_TESTNET:
+        case EndpointId.SOLANA_V2_MAINNET: {
+            const { chainName, env } = getNetworkForChainId(eid)
+            return `${chainName}-${env}`
+        }
+        default:
+            // For all other chains, we'll use the network name from hardhat.config.ts
+            return getNetworkNameForEid(eid)
+    }
+}
+
+const action: ActionType<TaskArgs> = async ({ logLevel = 'info', oappConfig }, hre) => {
+    setDefaultLogLevel(logLevel)
+    const logger = createLogger(logLevel)
+
+    const graph: OAppOmniGraph = await hre.run(SUBTASK_LZ_OAPP_CONFIG_LOAD, {
+        configPath: oappConfig,
+        schema: OAppOmniGraphHardhatSchema,
+        task: TASK_LZ_OAPP_CONFIG_GET,
+    } satisfies SubtaskLoadConfigTaskArgs)
+
+    const evmSdkFactory = createOAppFactory(createConnectedContractFactory())
+    const configs: Record<string, Record<string, unknown>> = {}
+
+    // Iterate over the graph of connections not from Solana
+    const tasks = graph.connections
+        .filter(({ vector: { from } }) => !isSolana(from))
+        .map(({ vector: { from, to } }) => async () => {
+            const endpointV2Sdk = await (await evmSdkFactory(from)).getEndpointSDK()
+
+            // OApp User Set Config
+            const receiveCustomConfig = await getReceiveConfig(endpointV2Sdk, to.eid, from.address, true)
+            const sendCustomConfig = await getSendConfig(endpointV2Sdk, to.eid, from.address, true)
+            const [sendCustomLibrary, sendCustomUlnConfig, sendCustomExecutorConfig] = sendCustomConfig ?? []
+            const [receiveCustomLibrary, receiveCustomUlnConfig] = receiveCustomConfig ?? []
```

Comment:
nit: might be helpful to add logs for if the config is a falsey value, same for below config checks